### ISSUE TITLE
Languages v2.1+ `/history` page out of date without notice

### ISSUE THREAD

**Issue by wackerow** [ethereum/ethereum-org-website#3543]
**Describe the bug**
Languages v2.1+ (French, Romanian) have history pages that are out of date, without notification of more recent content in English. 

**To Reproduce**

1. Go to https://ethereum.org/fr/history/
1. Note December 2020 is most recent timeline event, and there is no banner notifying there is a more updated version in English

**Expected behavior**

A TranslationBanner component should be used to identify this page as out of date and in need of translation help, and has a more recent version in English that could be viewed. 

**Screenshots**
![image](https://user-images.githubusercontent.com/54227730/127962894-ecce11a7-1dfe-43a5-ad2b-a1f1741f5d2b.png)

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by minimalsm** [comment]
Related PR: #3227

**Comment by samajammin** [comment]
Yes I think this will require changes to our gatsby-node.js logic to account for this page.

One problem: the updated English page isn't yet in Crowdin.

One broader question: should other markdown pages that have since been edited in English also display as outdated? e.g. https://ethereum.org/fr/developers/docs/gas/ is missing all the London upgrade info. A larger philosophical question is how much of a change before it becomes outdated, e.g. what if the English version has just fixed a small typo?
